extension/src/js/background/index.js

```diff
@@ -303,17 +319,21 @@ socket.on("peer joined", (peerData) => {
     sendState();
     startShare(peerData.id);
 });
+
 socket.on("peer ice", (iceData) => {
     console.log("Ice Candidate from peer: " + iceData.id + " in room: " + iceData.room);
     console.log("Ice Candidate: " + iceData.candidate);
-    if (roomID != iceData.room || !(iceData.id in peers)) {
+    if (roomID != iceData.room || !(iceData.id in peers) || (iceData.hostID !== socket.id)) {
```

Comment:
This is quite a complex if statement. What does it logically check for, and could we rip this out into a function?